src/components/CivicProfileForms/HousingInfo.jsx

```diff
@@ -47,48 +50,106 @@ const HousingInfo = () => {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form
+      onSubmit={handleSubmit}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '8px',
+        justifyContent: 'space-between'
+        // maxWidth: '720px',
+      }}
+    >
       <div
         style={{
-          padding: '8px',
           display: 'flex',
-          flexDirection: 'column',
           gap: '8px',
-          maxWidth: '360px'
+          minWidth: '846px',
+          padding: '8px'
+        }}
+      >
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '8px',
+            padding: '8px',
+            width: '40%'
+          }}
+        >
+          <TextField
+            id="street-input"
+            name="lastPermanentStreet"
+            label="Street:"
+            onChange={handleChange}
+            value={formData.lastPermanentStreet}
+          />
+          <TextField
+            id="city-input"
+            name="lastPermanentCity"
+            label="City:"
+            onChange={handleChange}
+            value={formData.lastPermanentCity}
+          />
+          <TextField
+            id="state-input"
+            name="lastPermanentState"
+            label="State:"
+            onChange={handleChange}
+            value={formData.lastPermanentState}
+          />
+          <TextField
+            id="zip-input"
+            name="lastPermanentZIP"
+            label="ZIP Code:"
+            onChange={handleChange}
+            value={formData.lastPermanentZIP}
+          />
+        </div>
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '8px',
+            padding: '8px',
+            width: '40%'
+          }}
+        >
+          <TextField
+            id="months-homeless-input"
+            name="monthsHomeless"
+            label="Months Houseless Past 3 Years:"
+            onChange={handleChange}
+            value={formData.monthsHomeless}
```

Comment:
Given it's already asking for number of months, think we could make the text field only accept numeric values.

It looks like MUI does have a custom component for Number Fields, but it looks like they're still unstable (https://mui.com/base-ui/react-number-input/)

The same could be said about number of times houseless.

Reply:
Have pushed to this branch, `513-Enhancement-Civic-Profile-Housing-Information-Form`, this change, with the three `monthsHomeless`, `timesHomeless`, and `timeToHousingLoss` input are now all `react-select` or `<Select>` fields with the correct `<MenuItem>` values in them from the HUD `hmis-interop` data values. Fixed.